pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
+
+`prompt off`
+
+- Download multiple files (glob expression):
+
+`mget {{*.png}}`
+
+- Upload multiple files (glob expression):
+
+`mput {{*.zip}}`
+
+- Delete multiple files on the remote machine (glob expression):
+
+`mdelete {{*.txt}}`
```

Comment:
Yes, multiple. But it works on single files too, so I basically never use the non-m variants. I'm not really sure why they bother with separate commands to be honest. But fair enough, I'll switch this with it's singular version.